app/src/main/java/sdp/moneyrun/Riddle.java

```diff
@@ -41,9 +46,53 @@ public String getAnswer() {
         return correctAnswer;
     }
 
+
+
+
+    // For some reason this method makes the DB
     public String[] getPossibleAnswers() {
         return new String[]{firstAnswer, secondAnswer, thirdAnswer, fourthAnswer};
     }
 
+    //////Needed for The DataBase Code
+    public String getCorrectAnswer() {
+        return correctAnswer;
+    }
+
+    public String getFirstAnswer() {
+        return firstAnswer;
+    }
+
+    public String getSecondAnswer() {
+        return secondAnswer;
+    }
+
+    public String getThirdAnswer() {
+        return thirdAnswer;
+    }
+
+    public String getFourthAnswer() {
+        return fourthAnswer;
+    }
+    //////////
+
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Needs to be tested

Reply:
Yep, added tests for both equals and hashcode